### PR TITLE
Fluff Augments

### PR DESCRIPTION
**Pull request by ToasterBan**
## About The Pull Request
Adds a small handful of augments inspired by baystation12s fluff augment system

https://github.com/DaedalusDock/daedalusdock/assets/54711687/2c937915-05c2-4035-b2a0-10d4667d7029

## Why It's Good For The Game
More customization is good

## Changelog

:cl:
add: New, non-gameplay changing augments in the character creator
/:cl:

### CODE REVIEW

File: code/__DEFINES/augments.dm [hunk 1]
```diff
@@ -19,6 +19,9 @@
 #define AUGMENT_SLOT_STOMACH		"Stomach"
 #define AUGMENT_SLOT_EYES			"Eyes"
 #define AUGMENT_SLOT_TONGUE			"Tongue"
+#define AUGMENT_SLOT_FLUFF_HEAD		"Minor Head Augment"
+#define AUGMENT_SLOT_FLUFF_CHEST	"Minor Chest Augment"
+#define AUGMENT_SLOT_FLUFF_GROIN	"Minor Groin Augment"
```
Comment (Kapu1178) [inline code/__DEFINES/augments.dm:24]: I don't like this but I also don't have any better ideas.

**Review by Kapu1178 — COMMENTED** [review]
As for specific augments, we can talk about it on discord.

**Comment by gonenoculer5** [comment]
@ToasterBan Could you add a list of the added fluff augments to the body of your PR so we can see what's been added in particular? (Yes, I know you can just look at the change feed but yeah)

**Comment by ToasterBan** [comment]
> @ToasterBan Could you add a list of the added fluff augments to the body of your PR so we can see what's been added in particular? (Yes, I know you can just look at the change feed but yeah)

They all have 0 mechanical functions, they technically can get EMPed but that does nothing at all

I just copy-pasted the names and descriptions of whatever bay had, excluding: Codex Access Chip, Neurostimulator, Pain assistant and the Data Chip because I arbitrarily considered them a bit too high tech for our lore

what we have is:
<details>
<summary>circadian conditioner </summary>

A small brain implant that carefully regulates the output of certain hormones to assist in controlling the sleep-wake cycle of its owner. May be an effective counter to insomnia, jet lag, and late-night work shifts.
</details>

<details>
<summary>genetic backup </summary>

This implant is a compact and resilient solid-state drive. It does nothing on its own, but contains the complete DNA sequence of its owner - whether it be to aid in medical treatment, serve for research purposes, or even be used as a template for vat-grown humans in the future.
</details>

<details>
<summary> emergency battery</summary>

A small emergency power supply. It provides no protection from power loss on its own, but might help keep your brain ticking through an otherwise critical failure.
</details>

<details>
<summary> skeletal bracing</summary>
Mechanical hinges and springs made from titanium or some other bio-compatible metal reinforce your joints, generally making strenuous activity less painful for you and allowing you to carry weight that would normally be unbearable. It provides no increase on strength on its own, unless you have weak bones to begin with.
</details>

<details>
<summary> ultraviolet shielding</summary>
Some parts of your epidermis have been replaced with a bio-engineered substitute that's resistant to harmful solar radiation - a common factor in the lives of spacers or inhabitants of planets with a weak magnetosphere."
</details>

<details>
<summary>recycler suite </summary>

In extreme environments where natural resources are limited or even nonexistent, it may be prudent to recycle nutrients and fluids the body would usually discard. This system reclaims any usable material in the digestive tract that would otherwise be lost to waste.
</details>

File: code/__DEFINES/augments.dm (hunk 1 above)
Comment (ToasterBan) [reply code/__DEFINES/augments.dm:24]: I originally was going to use the implant system but it turned out using that would have been much more scuffed

I don't know of any way off adding these without it being scuffed other than adding a completely new system, which would feel like bloating character customization

**Pull request merged**